### PR TITLE
[Dashboard] Remove chakra: BatchUpload drawer + other minor changes

### PR DESCRIPTION
**Pull request by kien-ngo**
Made the BatchUpload drawer more clean looking basically

---

## PR-Codex overview
This PR focuses on improving the UI components for the claim conditions form and batch lazy minting feature in a dashboard application. It includes updates to input handling, layout adjustments, and the introduction of a new sheet component for better user interaction.

### Detailed summary
- Updated `disabled` prop in `PhaseNameInput` component.
- Refactored layout in `ClaimConditions` component for better spacing.
- Replaced `Drawer` with `Sheet` in `BatchLazyMintButton` for enhanced UX.
- Improved state management for opening/closing modal in `BatchLazyMintButton`.
- Simplified form structure in `BatchLazyMint` component.
- Enhanced styling and responsiveness in `UploadStep` component.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
* **#5259** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5259?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/core-ui/batch-upload/upload-step.tsx [hunk 1]
```diff
@@ -20,117 +21,111 @@ export const UploadStep: React.FC<UploadStepProps> = ({
...
+            can add an <InlineCode code="image" /> and/or
+            <InlineCode code="animation_url" /> column and add the IPFS hashes
+            there.
+            <Link download color="blue.500" href="/example-with-ipfs.csv">
+              Download example.csv
+            </Link>
+          </li>
+          <li>
+            If you want to make your media files map to your NFTs, you can add
+            add the name of your files to the <InlineCode code="image" /> and
```
[the remark this answers is not in the document]
Comment (jnsdls) [reply apps/dashboard/src/core-ui/batch-upload/upload-step.tsx:121]: @kien-ngo this is a valid thing :)

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/core-ui/batch-upload/upload-step.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (kien-ngo) [reply apps/dashboard/src/core-ui/batch-upload/upload-step.tsx:121]: oops

File: apps/dashboard/src/core-ui/batch-upload/upload-step.tsx [hunk 2]
```diff
@@ -20,117 +21,111 @@ export const UploadStep: React.FC<UploadStepProps> = ({
...
-      <Container maxW="container.page">
-        <Flex gap={8} flexDir={{ base: "column", md: "row" }}>
-          <div className="relative aspect-square w-full md:w-1/2">
-            <div
+    <div className="flex flex-col items-center gap-8 md:flex-row">
+      <div className="relative aspect-square w-full md:w-1/2">
+        <div
+          className={cn(
+            "flex h-full cursor-pointer rounded-md border border-border hover:border-blue-500",
+            hasFailed ? "bg-red-200" : "bg-card",
```
Comment (jnsdls) [inline apps/dashboard/src/core-ui/batch-upload/upload-step.tsx:29]: there's no longer any container here, is that not required?

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/core-ui/batch-upload/batch-lazy-mint.tsx [hunk 3]
```diff
@@ -139,12 +137,10 @@ export const BatchLazyMint: ComponentWithChildren<BatchLazyMintProps> = (
...
 
   return (
-    <Container
-      maxW="container.page"
-      borderRadius={{ base: 0, md: "2xl" }}
-      my={{ base: 0, md: 12 }}
-      p={{ base: 0, md: 4 }}
-      as="form"
+    <form
+      className="mt-4 flex w-full flex-col"
```
[the remark this answers is not in the document]
Comment (kien-ngo) [reply apps/dashboard/src/core-ui/batch-upload/batch-lazy-mint.tsx:141]: The old design was too boxy (plus there are 2 scrollbars). I updated it to make it more clean. 
New: 



Old:

File: apps/dashboard/src/core-ui/batch-upload/upload-step.tsx (hunk 2 above)
Comment (kien-ngo) [reply apps/dashboard/src/core-ui/batch-upload/upload-step.tsx:29]: Same reason with comment above. I'm making change to this design hopefully you'd like it

**Review by jnsdls — APPROVED** [review]

**Pull request merged**